spec/unit/installer/target_installer/aggregate_target_installer_spec.rb

```diff
@@ -134,7 +134,8 @@ module Pod
         @installer.install!
       end
 
-      it 'creates a create copy resources script' do
+      # TODO: Remove this; the Copy Resources Script will soon be obsoleted by PR #3263
```

Comment:
That's a point where I asked for support. I'm not sure if the Copy Resources Script should be entirely removed yet. I believe it is still needed for frameworks and for bundles, but I need confirmation from someone actually understanding that wibbly wobbly stuff.